modules/calculators/Pid.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file Pid.h
+ * @brief PIDを計算するクラス
+ * @author miyahara046
+ */
+
+#ifndef PID_H
+#define PID_H
+
+// PIDゲインを保持する構造体
+struct PidGain {
+ public:
+  double kp;  // Pゲイン
+  double ki;  // Iゲイン
+  double kd;  // Dゲイン
+
+  /** コンストラクタ
+   * @param _kp Pゲイン
+   * @param _ki Iゲイン
+   * @param _kd Dゲイン
+   */
+  PidGain(double _kp, double _ki, double _kd);
+};
+
+class Pid {
+ public:
+  /** コンストラクタ
+   * @param _kp Pゲイン
+   * @param _ki Iゲイン
+   * @param _kd Dゲイン
+   * @param _targetValue 目標値
+   * @param _maxIntegral 累積積分値の最大値
+   * @param _minIntegral 累積積分値の最小値
+   */
+  Pid(double _kp, double _ki, double _kd, double _targetValue, double _maxIntegral,
+      double _minIntegral);
```

Comment:
maxIntegralとminIntegralってprivateの中で既に初期化してるから、コンストラクタでわざわざ初期化する必要はないと思う。
```suggestion
  Pid(double _kp, double _ki, double _kd, double _targetValue);
```

Reply:
コンストラクタをオーバーロードして複数作るといいと思います。

Reply:
```suggestion
  Pid(double _kp, double _ki, double _kd, double _targetValue, double _maxIntegral,
      double _minIntegral);
  Pid(double _kp, double _ki, double _kd, double _targetValue);
```
ちなみにコンストラクタのオーバーロードはこんな感じで，引数の違うコンストラクタを2つ作るってことです．